src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -37,9 +37,11 @@ type SummaryPanelEdgeState = {
 
 export default class SummaryPanelEdge extends React.Component<SummaryPanelPropType, SummaryPanelEdgeState> {
   static readonly panelStyle = {
-    width: '25em',
+    height: '100%',
+    margin: 0,
     minWidth: '25em',
-    overflowY: 'auto' as 'auto'
+    overflowY: 'auto' as 'auto',
```

Comment:
```
   'auto' as 'auto',
```
I don't understand why adding the `as 'auto'` part?

Reply:
There is a compile error if it's not casted.